docs/code_of_conduct.md

```diff
@@ -43,27 +43,32 @@ Project maintainers have the right and responsibility to remove, edit, or reject
 
 This section applies to RCOS meetings at RPI, including large group and small group meetings as well as bonus sessions, casual coding sessions, and any project meetings unless otherwise noted.
 
+### Attendance Codes
+Attendance codes are given during all RCOS small group, large group, and bonus sessions. When the attendance code is delivered, go to http://rcos.io, click "Attend", and enter the attendance code so you can be marked as present for the current session. 
+
+As much as we value sharing and openness, we also value integrity. Thus, we ask that you do not distribute attendance codes to people who are not currently attending the session. If you are caught distributing attendance codes to people who are absent, **both your attendance code as well as the attendance code(s) you distributed to others will be nullified**. 
```

Comment:
@AdrianCollado @Bad-Science This is the primary addition to the CoC - any objection to the following?

> If you are caught distributing attendance codes to people who are absent, **both your attendance code as well as the attendance code(s) you distributed to others will be nullified**. 

Realistically I don't think there's much we could do about nullifying the attendance of the individual who _shared_ the code, but I think it's worthy of inclusion to discourage such behavior. Thoughts?

Reply:
We could compromise by just nullifying the codes that were shared, making it kinda pointless to share them. However, that could be interpreted to mean there is no consequence for the person who shared them. We could say something along the lines of "the attendance codes you distributed to others will be nullified, and you may be subject to further consequences at the discretion of faculty" but the last part of that seems a bit scary. We could say repeat offenses may be subject to consequences at the discretion of faculty instead

Reply:
I think this is good. Even if we couldn't enforce it every time it's a good policy to have on the books.

Reply:
Speaking of that - the mentors and coordinators should probably do a better job enforcing the noise policy --- having it be universal should hopefully help so it'll become a norm